### DISCUSSION TITLE
How to reduce he length of the planned trajectory

### DISCUSSION THREAD

**Discussion by Entongsu** [NVlabs/curobo#252] [Software Q&A]
Hi,

I wonder how to reduce the length of the planned trajectory. I have found it too long some times. Thank you.

**Comment by balakumar-s** [comment]
What's the robot you are using? The trajectory can be long if you have small velocity limits for the joints (<0.5 radians/sec).

**Comment by peter-mitrano-bg** [comment]
Perhaps you're just asking about the number of time steps? i.e if you need to plan very short motions? I've used something like this:

```
        self.motion_gen_config: MotionGenConfig = MotionGenConfig.load_from_robot_config(
            ...
            trajopt_tsteps=24,
        )
```

Or you could reduce the `dt` used for each step, instead of reducing number of steps using some other arguments:
```
            trajopt_dt: Optional[float] = None,
            js_trajopt_dt: Optional[float] = None,
```
